spec/2-description.md

```diff
@@ -16,24 +14,24 @@ This specification has used several available and recognised open standards belo
 * [ISO/TS 17975:2015](https://www.iso.org/standard/61186.html) Health informatics — Principles and data requirements for consent in the Collection, Use or Disclosure of personal health information
 * [ISO/IEC TS 27560](https://www.iso.org/standard/80392.html) — Consent record information structure (under development)
 
-## 2.1 What Consent Is
+## What Consent Is
```

Comment:
@conradsp what's the guiding principle behind using section numbers vs. not using them?